api/ws/rpc/wsRPC.js

```diff
@@ -182,7 +187,7 @@ ClientRPCStub.prototype.sendAfterSocketReadyCb = function (connectionState) {
 						return setImmediate(cb, null, res);
 					})
 					.catch(function (err) {
-						return setImmediate(cb, err);
+						return setImmediate(cb, err.toString());
```

Comment:
because of https://github.com/LiskHQ/lisk/pull/734/files#diff-f916757f1eba9ea0e2bf3f2ce114ebcdR56